docs/user-guide/cli.md

```diff
@@ -567,20 +564,21 @@ This estimates fees for a transaction that sends 22 lovelace to `Ae2tdPwUPEZ...n
 
 :::{.cli}
 ```
-cardano-wallet transaction list [--port INT] WALLET_ID [--start TIME] [--end TIME] [--order ORDER]
+cardano-wallet transaction list [--port INT] WALLET_ID [--start TIME] [--end TIME] [--order ORDER] [--simple-metadata] [--max_count MAX_COUNT]
 ```
 :::
 
-List all incoming and outgoing transactions for the wallet:
+List the transactions associated with a wallet.
 
 ```
 $ cardano-wallet transaction list 2512a00e9653fe49a44a5886202e24d77eeb998f \
     --start 2018-09-25T10:15:00Z \
     --end 2019-11-21T10:15:00Z \
-    --order ascending
+    --order ascending \
+    --max_count 10
 ```
 
-This lists all transactions between `2018-09-25T10:15:00Z` and `2019-11-21T10:15:00Z` in `ascending` order.
+This lists max 10 transactions between `2018-09-25T10:15:00Z` and `2019-11-21T10:15:00Z` in `ascending` order.
```

Comment:
how do we document the fact that without `max_count ` they will be all included? I mean it's obvious ...